Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the moco-controller binary
-FROM --platform=$BUILDPLATFORM quay.io/cybozu/golang:1.19-focal as builder
+FROM --platform=$BUILDPLATFORM ghcr.io/cybozu/golang:1.21-jammy as builder
```

Comment:
Please use the same varion of Ubuntu for the build stage, mysql:8.0.32.2 and the backup image.

```
# For MySQL binaries
FROM --platform=$TARGETPLATFORM ghcr.io/cybozu-go/moco/mysql:8.0.32.2 as mysql

# the backup image
FROM --platform=$TARGETPLATFORM quay.io/cybozu/ubuntu:20.04
```